src/createAction.js

```diff
@@ -8,12 +8,19 @@ export default function createAction(type, actionCreator, metaCreator) {
     : identity;
 
   return (...args) => {
-    const action = {
+    let action = {
```

Comment:
To me it's less about the semantics of the keyword and more about consistency with the rest of the code base. `const` is used whenever a variable does not need to be reassigned.